README.md

```diff
@@ -15,14 +15,43 @@ The image uses the following software:
+First, decide whether you want your container to be part of the public, production Stellar network (referred to as the _pubnet_) or the test network (called testnet) that we recommend you use while developing software because you need not worry about losing money on the testnet. Additionally, we have added a standalone network (called standalone) which allows you to run your own private Stellar network.
 
 Next, you must decide whether you will use a docker volume or not.  When not using a volume, we say that the container is in _ephemeral mode_, that is, nothing will be persisted between runs of the container. _Persistent mode_ is the alternative, which should be used in the case that you need to either customize your configuration (such as to add a validation seed) or would like avoid a slow catchup to the Stellar network in the case of a crash or server restart.  We recommend persistent mode for anything besides a development or test environment.
 
 Finally, you must decide what ports to expose.  The software in these images listen on 4 ports, each of which you may or may not want to expose to the network your host system is connected to.  A container that exposes no ports isn't very useful, so we recommend at a minimum you expose the horizon http port.  See the "Ports" section below for a more nuanced discussion regarding the decision about what ports to expose.
 
 After deciding on the questions above, you can setup your container.  Please refer to the appropriate section below based upon what mode you will run the container in.
 
+### Network Options
+
+Provide either `--pubnet`, `--testnet` or `--standalone` as a command line flag when starting the container to determine which network (and base configuration file) to use.
+
+#### `--pubnet`
+
+In public network mode, the node will join the public, production Stellar network.
+
+#### `--testnet`
+
+In test network mode, the node will join the network that developers use while developing software. Use the [Stellar Laboratory](https://laboratory.stellar.org/#account-creator?network=test) to create an account on the test network.
+
+#### `--standalone`
+
+In standalone network mode, you can optionally pass `--protocol-version {version}` parameter to run a specific protocol version (defaults to latest version).
+
+The network passphrase of the network defaults to:
+```
+Standalone Network ; February 2017
```

Comment:
I don't know where this network passphrase comes from but the core standalone config is not that one (since 2015) - see https://github.com/stellar/stellar-core/blob/master/docs/stellar-core_standalone.cfg

Reply:
I can see this old passphrase in three different places in this repo:

https://github.com/stellar/docker-stellar-core-horizon/blob/4b6988eb6c3d7cb6dc2c841b04120eba4a3e719f/start#L98

https://github.com/stellar/docker-stellar-core-horizon/blob/23e8df8df413f5d2eb5030f511367a2f3cc037c3/standalone/core/etc/stellar-core.cfg#L9

https://github.com/stellar/docker-stellar-core-horizon/blob/23e8df8df413f5d2eb5030f511367a2f3cc037c3/standalone/core/etc/stellar-captive-core.cfg#L2

Should we replace those with the one in the stellar-core repo?

Reply:
I think we weigh the value in changing the network passphrase for the standalone network in this image because that would be a breaking change to anyone experimenting with it that would probably not be obvious and be quite time consuming to debug and discover.